### ISSUE TITLE
Unable to schedule posts after updating to Lemmy v0.19.1

### ISSUE THREAD

**Issue by etymotic** [RikudouSage/LemmySchedule#23]
Originally scheduled post did not post after upgrading Lemmy to 0.19.1 (we skipped 0.19.0).

Deleting cache and uploads folders did not fix the problem, and I am not able to schedule any posts. There are errors in the logs about not being able to fetch communities.
```

 // Warming up the cache for the prod environment with debug false              

 [OK] Cache for the "prod" environment (debug=false) was successfully warmed.   

Unlinking stale socket /var/run/supervisor.sock
Starting supervisor: supervisord.
No config updates to processes

 [OK] Consuming messages from transport "async".                                

 // The worker will automatically exit once it has processed 10 messages,       
 // exceeded 256M of memory, been running for 3600s or received a stop signal   
 // via the messenger:stop-workers command.                                     

 // Quit the worker with CONTROL-C.                                             


 [OK] Consuming messages from transport "async".                                

 // The worker will automatically exit once it has processed 10 messages,       
 // exceeded 256M of memory, been running for 3600s or received a stop signal   
 // via the messenger:stop-workers command.                                     

 // Quit the worker with CONTROL-C.                                             

AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.20.0.2. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.20.0.2. Set the 'ServerName' directive globally to suppress this message
[Thu Dec 21 13:40:31.962160 2023] [core:warn] [pid 39] AH00098: pid file /var/run/apache2/apache2.pid overwritten -- Unclean shutdown of previous Apache run?
[Thu Dec 21 13:40:31.967454 2023] [mpm_prefork:notice] [pid 39] AH00163: Apache/2.4.57 (Debian) PHP/8.2.13 configured -- resuming normal operations
[Thu Dec 21 13:40:31.967626 2023] [core:notice] [pid 39] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
172.20.0.1 - - [21/Dec/2023:13:41:35 +0000] "GET / HTTP/1.1" 302 709 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
172.20.0.1 - - [21/Dec/2023:13:41:36 +0000] "GET /auth/login HTTP/1.1" 200 2504 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
[info] Received message App\Job\RefreshInstanceListJob
[info] Request: "GET https://data.lemmyverse.net/data/instance.min.json"
[info] Response: "200 https://data.lemmyverse.net/data/instance.min.json"
[info] Sending message App\Job\RefreshInstanceListJob with async sender using Symfony\Component\Messenger\Bridge\Redis\Transport\RedisTransport
[info] Message App\Job\RefreshInstanceListJob handled by App\JobHandler\RefreshInstanceListHandler::__invoke
[info] App\Job\RefreshInstanceListJob was handled successfully (acknowledging to transport).
172.20.0.1 - - [21/Dec/2023:13:41:37 +0000] "GET /build/fonts/fa-solid-900.1551f4f6.woff2 HTTP/1.1" 200 78521 "https://schedule.wetshaving.social/build/app.8d3c59f8.css" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
172.20.0.1 - - [21/Dec/2023:13:43:25 +0000] "POST /auth/login HTTP/1.1" 302 1206 "https://schedule.wetshaving.social/auth/login" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
[info] Received message App\Job\FetchCommunitiesJob
[info] Request: "GET https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[info] Response: "200 https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[warning] Error thrown while handling message App\Job\FetchCommunitiesJob. Sending for retry #1 using 1000 ms delay. Error: "Handling "App\Job\FetchCommunitiesJob" failed: Rikudou\LemmyApi\Response\Aggregates\CommunityAggregates::__construct(): Argument #3 ($hotRank) not passed"
[info] Received message App\Job\FetchCommunitiesJob
[info] Request: "GET https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[info] Response: "200 https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[warning] Error thrown while handling message App\Job\FetchCommunitiesJob. Sending for retry #2 using 2000 ms delay. Error: "Handling "App\Job\FetchCommunitiesJob" failed: Rikudou\LemmyApi\Response\Aggregates\CommunityAggregates::__construct(): Argument #3 ($hotRank) not passed"
[info] Received message App\Job\FetchCommunitiesJob
[info] Request: "GET https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[info] Response: "200 https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[warning] Error thrown while handling message App\Job\FetchCommunitiesJob. Sending for retry #3 using 4000 ms delay. Error: "Handling "App\Job\FetchCommunitiesJob" failed: Rikudou\LemmyApi\Response\Aggregates\CommunityAggregates::__construct(): Argument #3 ($hotRank) not passed"
[info] Received message App\Job\FetchCommunitiesJob
[info] Request: "GET https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[info] Response: "200 https://sub.wetshaving.social/api/v3/community/list?limit=20&page=1&sort=TopAll&type_=All&show_nsfw=true&auth=redacted"
[critical] Error thrown while handling message App\Job\FetchCommunitiesJob. Removing from transport after 3 retries. Error: "Handling "App\Job\FetchCommunitiesJob" failed: Rikudou\LemmyApi\Response\Aggregates\CommunityAggregates::__construct(): Argument #3 ($hotRank) not passed"
```

**Comment by RikudouSage** [comment]
That should be fixed in 1.5.3, I've fixed it yesterday in dev branch and released it as 1.5.3 few minutes ago. Do let me know if it works for you.

**Comment by etymotic** [comment]
Scheduling a new post successfully lists the communities, but throws the 500 Internal Error.

```
172.20.0.1 - - [22/Dec/2023:20:12:14 +0000] "GET /post/list HTTP/1.1" 200 2393 "https://schedule.wetshaving.social/post/list" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
172.20.0.1 - - [22/Dec/2023:20:12:14 +0000] "GET /post/list HTTP/1.1" 200 2393 "https://schedule.wetshaving.social/post/list" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
172.20.0.1 - - [22/Dec/2023:20:12:16 +0000] "GET /post/create HTTP/1.1" 200 8700 "https://schedule.wetshaving.social/post/list" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
172.20.0.1 - - [22/Dec/2023:20:12:16 +0000] "POST /en/_components/ScheduleComponent/setTimezoneAsString HTTP/1.1" 200 1869 "https://schedule.wetshaving.social/post/create" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
172.20.0.1 - - [22/Dec/2023:20:12:20 +0000] "POST /post/ajax/title/expression HTTP/1.1" 200 372 "https://schedule.wetshaving.social/post/create" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
172.20.0.1 - - [22/Dec/2023:20:12:42 +0000] "GET /en/_components/ScheduleComponent?props=%7B%22amount%22%3A1%2C%22scheduleType%22%3A0%2C%22timeZone%22%3A%22America%2FNew_York%22%2C%22weekday%22%3A1%2C%22targetTime%22%3Anull%2C%22selectedDayType%22%3A0%2C%22day%22%3A1%2C%22%40attributes%22%3A%7B%22data-live-id%22%3A%22live-2968125594-0%22%7D%2C%22%40checksum%22%3A%22SAWq1gFhuGiKRecUsWW3eFoCyhnDUl0T0hOTg8RVzE0%3D%22%7D&updated=%7B%22scheduleType%22%3A%220%22%2C%22targetTime%22%3A%2200%3A00%22%7D HTTP/1.1" 200 2635 "https://schedule.wetshaving.social/post/create" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
[Fri Dec 22 20:13:03.627591 2023] [php:notice] [pid 45] [client 172.20.0.1:37638] [critical] Uncaught PHP Exception Error: "Unknown named parameter $federationSignedFetch" at /var/www/html/vendor/rikudou/lemmy-api/src/Response/AbstractResponseDto.php line 104, referer: https://schedule.wetshaving.social/post/create
172.20.0.1 - - [22/Dec/2023:20:13:03 +0000] "POST /post/create/do HTTP/1.1" 500 1353 "https://schedule.wetshaving.social/post/create" "Mozilla/5.0 (X11; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0"
```

**Comment by RikudouSage** [comment]
Is there some test community on your instance I might use to test this? My instance is still on 0.18.5.

**Comment by etymotic** [comment]
[!test@sub.wetshaving.social](https://sub.wetshaving.social/c/test)

**Comment by etymotic** [comment]
Still having this problem after updating to Lemmy v0.19.2.

**Comment by RikudouSage** [comment]
Should be fixed in version 1.6.0.

**Comment by etymotic** [comment]
> Should be fixed in version 1.6.1.

Working now. Thank you!
